File: src/com/l2kt/gameserver/network/serverpackets/ShowMiniMap.java
package com.l2kt.gameserver.network.serverpackets;

import com.l2kt.gameserver.instancemanager.SevenSigns;

public class ShowMiniMap extends L2GameServerPacket
{
	public static final ShowMiniMap REGULAR_MAP = new ShowMiniMap(1665);
	
	private final int _mapId;
	
	public ShowMiniMap(int mapId)
	{
		_mapId = mapId;
	}
	
	@Override
	protected final void writeImpl()
	{
		writeC(0x9d);
		writeD(_mapId);
		writeD(SevenSigns.getInstance().getCurrentPeriod().ordinal());
	}
}

File: src/com/l2kt/gameserver/network/serverpackets/ShowMiniMap.kt
package com.l2kt.gameserver.network.serverpackets

import com.l2kt.gameserver.instancemanager.SevenSigns

class ShowMiniMap(private val _mapId: Int) : L2GameServerPacket() {

    override fun writeImpl() {
        writeC(0x9d)
        writeD(_mapId)
        writeD(SevenSigns.currentPeriod.ordinal)
    }

    companion object {
        val REGULAR_MAP = ShowMiniMap(1665)
    }
}